conf/test.config

```diff
@@ -19,7 +19,9 @@ params {
     max_memory = '6.GB'
     max_time   = '6.h'
 
+
     // Input data
+    // params.test_data_base (default) = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq3/'
```

Comment:
where are the rests of the comments for the other files?

Reply:
I summarised it in a single comment there because we can still use this to extract the default path to the test data rather than having one URL per line in the file.